lib/puppet_x/puppetlabs/scheduled_task/task.rb

```diff
@@ -192,8 +192,8 @@ class TASK_STATE
       TASK_STATE_RUNNING    = 4
     end
 
-    # https://docs.microsoft.com/en-us/windows/win32/api/taskschd/ne-taskschd-task_instances_policy
-    class TASK_INSTANCES_POLICY
+    # https://docs.microsoft.com/en-us/windows/win32/api/taskschd/ne-taskschd-multiple_instances
```

Comment:
This link is broken. Also, let's not change this TASK_INSTANCES_POLICY enumeration class yet since it's not being used in our code.